src/tests/e2e/mocks/cryptoAccounts.js

```diff
@@ -104,7 +104,7 @@ const CRYPTO_ACCOUNTS = [
   // ledger
   {
     addedAt: 1583802157,
-    address: '0xd3ced3b16c8977ed0e345d162d982b899e978588',
+    address: '0x6F6a68eC176AAfFE76e6771F2Ea79607f6B79801',
```

Comment:
What's the reason of making this change.

Reply:
bypass 0xd3ced3b16c8977ed0e345d162d982b899e978588 for allowance setting for metamask, but we do not want to bypass other types of wallets. This further reduced the number of calls.